### ISSUE TITLE
help wanted 

### ISSUE THREAD

**Issue by Philepic** [Philepic/social-network-in-php#1]
guys i wanna make a project in php like souncloud and i dont know how to start can anybody help me please

**Comment by Jaylooker** [comment]
* This is a big project considering it's a website + database so you should probably follow the SDLC (Software Development LifeCycle). This means defining **valid** requirements, modeling some things like a BA(Business Analyst) to keep sane, and testing. By following the SDLC you're going to need follow a methodogy (Scrum, Agile, Waterfall...)
* You're going need a database to store and retrieve the files (MySQL, sqlite, ...). 
* You're also going to need to find what metadata you want to store about each song and how you want to store it (JSON, XML, YAML, ...)
* You're going need a find place to host your servers (AWS service, IBM service, Google service...) and the website (Github Pages, BlueHost, HostGator, ...). If you want custom domain you're going to need to pick a domain service.
* You're going to need to pick what libraries (handling data format, ...) and frameworks (PHP-related, ...) you intend to use. 
* It's probably going to be a dynamic website. 
* You're also going to need some security throughout the system 
* There's also some things to consider about maintaining the code and contributions
* There are some things I'm probably omitting but are important to consider...

I found some applications developed in PHP that are very similar to what you're wanting to do:
[Ampache](https://github.com/ampache/ampache)
[Koel](https://github.com/phanan/koel)

Also a stackoverflow [question](https://stackoverflow.com/questions/4965756/how-to-develop-a-music-streaming-website-using-php) of relevance.

TLDR
* Need to answer a lot of questions
* Need a precise plan and scope of project (how big) 
* Need a website
* Need database servers
* Need a host
* Then can launch
